changelog/databases/_posts/2020-11-16-postgresql-12.5.0-1.markdown

```diff
@@ -0,0 +1,18 @@
+---
+modified_at: 2020-11-16 11:00:00
+title: 'PostgreSQL - New Scalingo release: 12.5.0-1, 11.10.0-1, 10.15.0-1 and 9.6.20'
+---
+
+New default version: **12.5.0-1**
+
+Changelog:
+- Bump version of PostgreSQL to last minor. It contains a security fix related
+  to a CVE:
+    - [CVE-2020-25694](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-25694)
```

Comment:
Actually in contains fixes for several CVE, maybe linking https://www.postgresql.org/about/news/postgresql-131-125-1110-1015-9620-and-9524-released-2111/ is better?

Reply:
And mentioning all databases will be updated in the coming days